packages/puppeteer/package.json

```diff
@@ -0,0 +1,138 @@
+      "require": "./lib/cjs/puppeteer/puppeteer.js"
+    },
+    "./internal/*": {
+      "import": "./lib/esm/puppeteer/*",
+      "require": "./lib/cjs/puppeteer/*"
+    },
+    "./*": {
+      "import": "./*",
+      "require": "./*"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/puppeteer/puppeteer/tree/main/packages/puppeteer"
+  },
+  "publishConfig": {
+    "registry": "https://wombat-dressing-room.appspot.com"
+  },
+  "engines": {
+    "node": ">=14.1.0"
+  },
+  "scripts": {
+    "build:tsc": "wireit",
+    "build:types": "wireit",
+    "build": "wireit",
+    "clean": "tsc -b --clean && rimraf lib",
+    "format:types": "wireit",
+    "generate:package-json": "wireit",
+    "generate:sources": "wireit",
+    "postinstall": "node install.js",
+    "postpack": "tsx ../../utils/insert_file_dependencies.ts ./package.json",
+    "prepack": "tsx ../../utils/resolve_file_dependencies.ts ./package.json"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "format:types",
+        "generate:package-json"
+      ]
+    },
+    "generate:sources": {
+      "command": "tsx tools/generate_sources.ts",
+      "dependencies": [
+        "../puppeteer-core:build"
+      ],
+      "files": [
+        "tools/generate_sources.ts"
+      ],
+      "output": [
+        "src/types.ts"
+      ]
+    },
+    "generate:package-json": {
+      "command": "echo '{\"type\": \"module\"}' > lib/esm/package.json",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "build:tsc"
+      ],
+      "output": [
+        "lib/esm/package.json"
+      ]
+    },
+    "build:types": {
+      "command": "api-extractor run --local",
+      "dependencies": [
+        "build:tsc"
+      ],
+      "files": [
+        "tsconfig.json",
+        "api-extractor.json",
+        "lib/esm/puppeteer/types.d.ts"
+      ],
+      "output": [
+        "lib/types.d.ts"
+      ]
+    },
+    "format:types": {
+      "command": "eslint --cache-location .eslintcache --cache --ext=ts --no-ignore --no-eslintrc -c=../../.eslintrc.types.cjs --fix lib/types.d.ts",
+      "dependencies": [
+        "build:types"
+      ],
+      "clean": false,
+      "files": [
+        "lib/types.d.ts",
+        "../../.eslintrc.types.cjs"
+      ],
+      "output": [
+        "lib/types.d.ts"
+      ]
+    },
+    "build:tsc": {
+      "command": "tsc -b",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "../puppeteer-core:build",
+        "generate:sources"
+      ],
+      "files": [
+        "src/**",
+        "compat/**",
+        "**/tsconfig.*.json"
+      ],
+      "output": [
+        "lib/**"
+      ]
+    }
+  },
+  "files": [
+    "lib",
+    "install.js",
+    "!*.tsbuildinfo"
+  ],
+  "author": "The Chromium Authors",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "https-proxy-agent": "5.0.1",
+    "progress": "2.0.3",
+    "proxy-from-env": "1.1.0",
+    "puppeteer-core": "file:../puppeteer-core"
```

Comment:
I thought we should be able to use a version here and NPM would be smart enough to link it automatically and during the release the version can be updated?

Reply:
Not sure release-please would automate this. `release-please` wasn't built with this in mind since lerna originally did all the versioning. We are basically doing that specific part of lerna here.